module/Activity/src/Activity/Controller/ActivityController.php

```diff
@@ -7,6 +7,7 @@
 use Zend\Mvc\Controller\AbstractActionController;
 use Zend\Session\Container as SessionContainer;
 use Activity\Form\ActivitySignup as SignupForm;
+use Activity\Form\ModifyRequest as RequestForm;
```

Comment:
This form does not seem to exist? Also it is better to retrieve new form instances from the service using the service manager. (To allow easy passing of constructors for translation, etc.)